src/otx/algo/keypoint_detection/heads/rtmcc_head.py

```diff
@@ -0,0 +1,245 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
It seems that many files include mmlab copyright. could you update them?

Reply:
Sungchul told me that I should include them if the codes are come from openmmlab. @sungchul2, could you confirm this?

Reply:
As far as I know, if the source code is used as is, the license should remain as is as well.
But it seems that there are additional importing modules and modified type annotations, so updated license seems required.